config.json

```diff
@@ -2,7 +2,7 @@
     "website_name": "Binary.com",
     "display_languages": "EN,ID,RU,ES,FR,PT,DE,ZH_CN,PL,AR",
     "static": {
-        "url": "https://static.binary.com/",
+        "url": "https://negar-binary.github.io/binary-static/",
```

Comment:
Noooooo :)

Use 
git update-index --assume-unchanged config.json
And you will never be asked to commit changes to this file again.